bin/install-osx-dev.sh

```diff
@@ -80,7 +80,7 @@ fi
 
 
 # Install testing dependencies
-pip install -r "test/integration_tests/requirements.txt"
+pip install -r "test/integration_tests/requirements-integration-test.txt"
```

Comment:
@ryansanford Do we really need "integration_test" twice in the path? Doesn't that defeat the purpose of subdirectories?